packages/components/src/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ReactElement, ReactNode } from "react";
+import styles from "./RadioGroup.css";
+
+interface RadioGroupProps {
+  readonly children: ReactElement | ReactElement[];
+  readonly value: string;
```

Comment:
Would be good to describe these props. Eg:

https://github.com/GetJobber/atlantis/blob/master/packages/components/src/FeatureSwitch/FeatureSwitch.tsx#L13

Reply:
Value may also be a `number`.

```suggestion
  readonly value: string | number;
```